pynance/data/retrieve.py

```diff
@@ -10,24 +10,29 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
```

Comment:
This line should be removed, and I think the space is important for generating the documentation.